youtube_dl/extractor/servus.py

```diff
@@ -5,26 +5,24 @@
 
 
 class ServusIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?servus\.com/(?:at|de)/p/[^/]+/(?P<id>AA-\w+|\d+-\d+)'
+    _VALID_URL = r'https?://(?:www\.)?servus\.com/tv/videos/(?P<id>aa-\w+|\d+-\d+)'
     _TESTS = [{
-        'url': 'https://www.servus.com/de/p/Die-Gr%C3%BCnen-aus-Sicht-des-Volkes/AA-1T6VBU5PW1W12/',
+        'url': 'https://www.servus.com/tv/videos/aa-1t6vbu5pw1w12/',
         'md5': '046dee641cda1c4cabe13baef3be2c1c',
         'info_dict': {
-            'id': 'AA-1T6VBU5PW1W12',
+            'id': 'aa-1t6vbu5pw1w12',
```

Comment:
Ids must stay exactly the same.

Reply:
The ID is part of the link, and is the same

Reply:
`AA-1T6VBU5PW1W12` != `aa-1t6vbu5pw1w12`.

Reply:
Ok i fixed the test, sorry for the question, now it is clear!